src/style.css

```
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo {
+  padding-top: 3px;
+}
+
+.nav-menu {
+  display: flex;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav-list {
+  display: block;
+  list-style: none;
+}
+
+.nav-link {
+  color: #000;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 14.06px;
```

```suggestion
  line-height: 14.06px;
```
It's a very strange number. try to avoid using such `magic` numbers in your code. You may center your text vertically by display:flex, align-items-center and specify height for it